frontend/integration-tests/tests/event.scenario.ts

```diff
@@ -0,0 +1,56 @@
+import { browser, $, element, by, ExpectedConditions as until } from 'protractor';
+import { appHost, checkLogs, checkErrors, testName } from '../protractor.conf';
+import { safeLoad, safeDump } from 'js-yaml';
+import * as _ from 'lodash';
+import * as crudView from '../views/crud.view';
+import * as yamlView from '../views/yaml.view';
+
+describe('Events', () => {
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  describe('Events', () => {
+    const name = `${testName}-pod`;
+    const testLabel = 'automatedTestName';
+
+    it('displays Events all resources', async () => {
+      await browser.get(`${appHost}/k8s/all-namespaces/events`);
+      await browser.wait(until.presenceOf($('.co-sysevent__box')));
+      expect($('.co-sysevent__box').isPresent()).toBe(true);
+    });
+
+    it('creates a new pod resource instance for event testing', async () => {
+      await browser.get(`${appHost}/k8s/ns/${testName}/pods`);
+      await crudView.isLoaded();
+      await crudView.createYAMLButton.click();
+      await yamlView.isLoaded();
+      const content = await yamlView.getEditorContent();
+      const newContent = _.defaultsDeep(
+        {},
+        { metadata: { name, labels: { [testLabel]: testName } } },
+        safeLoad(content),
+      );
+      await yamlView.setEditorContent(safeDump(newContent));
+      await yamlView.saveButton.click();
+      expect(crudView.errorMessage.isPresent()).toBe(false);
+    });
+
+    it('event view displays created pod', async () => {
+      await browser.get(`${appHost}/ns/${testName}/events`);
+      await browser.wait(until.presenceOf($('.co-sysevent-stream__status')), 10000);
+      await browser.wait(until.presenceOf($('.co-sysevent__box')), 10000);
+      await browser.wait(until.presenceOf(element(by.linkText(name))), 10000);
+      await browser.wait(until.presenceOf($(`[data-test-id=${name}]`)));
```

Comment:
Do we need to wait for each of these elements? I'd think we could have only this last wait.

In most cases, we can just use the default timeout. It's unusual you would need something different.